src/APlayer.js

```diff
@@ -846,6 +846,66 @@ class APlayer {
         }
     }
 
+      /**
+    * Remove song from playlist 
+    */
+    removeSong(indexOfSong) {
+        
+               if (this.option.music[indexOfSong] != null) { // Check if song exists 
+                   const list = this.element.getElementsByClassName('aplayer-list')[0];
+                   var oList = list.firstElementChild; // OL tag
+                   var liList = []; // Holds the index of the LI tags
```

Comment:
Why do this, `oList.getElementsByTagName('li')` is more straightforward

Reply:
That's a lot easier, thanks. Change made.